src/metadataparsers/MetadataParser.php

```diff
@@ -43,4 +44,15 @@ public function __construct($settings)
         $this->logStreamHandler= new \Monolog\Handler\StreamHandler($this->pathToLog, Logger::WARNING);
         $this->log->pushHandler($this->logStreamHandler);
     }
+
+    /**
+     * Gets the metadata for a specific object.
+     *
+     * @param string $record_key
+     *   The object's record key.
+     *
+     * @return string
+     *   The object's metadata.
+     */
+    abstract public function metadata($record_key);
```

Comment:
So alot of `phpcbf` cleaning up the code, but otherwise made this `abstract` and added the abstract definition for the `metadata` function.